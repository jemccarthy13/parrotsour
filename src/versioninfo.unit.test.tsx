import { mount } from "enzyme"
import React from "react"
import VersionInfo from "./versioninfo"

describe("VersionInfo", () => {
  it("has_version", () => {
    const wrapper = mount(<VersionInfo />)
    const vInfo = wrapper.find("#vInfo")
    expect(vInfo).toHaveLength(1)
    expect(vInfo.text().indexOf("Version")).not.toEqual(-1)
    expect(vInfo).toMatchSnapshot()
  })
})
