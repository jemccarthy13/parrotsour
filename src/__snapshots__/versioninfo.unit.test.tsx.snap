// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VersionInfo has_version 1`] = `ReactWrapper {}`;
